avldb/src/main/scala/scorex/db/LDBKVStore.scala

```diff
@@ -27,6 +27,25 @@ class LDBKVStore(protected val db: DB) extends KVStoreReader with ScorexLogging
     }
   }
 
+  /**
+    * Insert single key-value into database
+    * @param id - key to insert
+    * @param value - value to insert
+    * @return - Success(()) in case of successful insertion, Failure otherwise
+    */
+  def insert(id: K,  value: V): Try[Unit] = {
+    val batch = db.createWriteBatch()
```

Comment:
Whydo we construct batch here? Why not to insert record to the storage directly - using insert call?

Reply:
Indeed. Fixed